cmd/serve.go

```diff
@@ -33,7 +33,7 @@ var (
 				cfg.DataDir = serverDataDir
 			}
 			sp := fmt.Sprintf("%s/.ssh", cfg.DataDir)
-			kp, err := keygen.NewWithWrite(sp, "charm_server", []byte(""), keygen.RSA)
+			kp, err := keygen.NewWithWrite(sp, "charm_server", []byte(""), keygen.Ed25519)
```

Comment:
wont this trigger SSH key check errors on existing servers?

e.g.: there was a server using a RSA key, and now its a ED25519 key, which is different from the previous one...

Reply:
There's no host key verification in `charm` so it _should_ be ok.